### PR TITLE
Add `FlightDisplay` + some QoL improvements

### PR DESCRIPTION
**Pull request by harshil21**
Notable changes:

- Use a class to handle displays
- Flight data is not lost upon Ctrl+C
- Mock sim auto terminates when flight data is exhausted
- Some more banners

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -22,11 +22,13 @@
     SIMULATION_LOG_PATH,
     UPSIDE_DOWN,
 )
-from utils import prepare_process_dict, update_display
+from utils import FlightDisplay
```
Comment (JacksonElia) [inline main.py:25]: if we're making a whole class for this, I think we should move it to `mock/`

File: utils.py [hunk 2]
```diff
@@ -45,66 +53,86 @@ def deadband(input_value: float, threshold: float) -> float:
...
-    all_processes = {}
-    imu_process = airbrakes.imu._data_fetch_process
-    log_process = airbrakes.logger._log_process
-    current_process = multiprocessing.current_process()
-    for p in [imu_process, log_process, current_process]:
-        all_processes[p.name] = psutil.Process(p.pid)
-    return all_processes
+
+    # Initialize Colorama
+    init(autoreset=True)  # Automatically reset colors after each print
```
Comment (JacksonElia) [inline utils.py:62]: shouldn't this be in the init?

**Review by JacksonElia — CHANGES_REQUESTED** [review]
Overall looks great! Biggest change would be moving it to mock I think

**Comment by JacksonElia** [comment]
I don't think you committed the change of actually moving it to mock? It isn't in mock.

**Comment by harshil21** [comment]
oops

File: airbrakes/mock/display.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    def prepare_process_dict(self) -> dict[str, psutil.Process]:
+        """
+        Prepares a dictionary of processes to monitor CPU usage for.
+        """
+        all_processes = {}
+        imu_process = self.airbrakes.imu._data_fetch_process
+        log_process = self.airbrakes.logger._log_process
+        current_process = multiprocessing.current_process()
+        for p in [imu_process, log_process, current_process]:
+            all_processes[p.name] = psutil.Process(p.pid)
```
Comment (JacksonElia) [inline airbrakes/mock/display.py:103]: I'm gonna nit pick you here, method needs :return in docstring and maybe add a comment for this line for psutil

Comment (harshil21) [reply airbrakes/mock/display.py:103]: what do you want the comment to tell? Also, I love nitpicks.

Comment (JacksonElia) [reply airbrakes/mock/display.py:105]: probably what `psutil.Process(p.pid)` does -- someone unfamiliar to what is happening might get confused

**Review by JacksonElia — APPROVED** [review]
LGTM

**Pull request merged**